verify/v1/verify.yaml

```diff
@@ -0,0 +1,808 @@
+      type: object
+      properties:
+        is_in:
+          type: string
+          example: "mobile, fixed"
+          description: phone number type IS IN mobile or fixed.
+    OperatorCountryIsCriteriaItem:
+      type: object
+      properties:
+        is:
+          type: string
+          example: "DEU"
+          description: number IS in DEU.
+    OperatorCountryIsInCriteriaItem:
+      type: object
+      properties:
+        is_in:
+          type: string
+          example: "NOR,SWE,FIN"
+          description: number IS IN these countries.
+    MccMncIsCriteriaItem:
+      type: object
+      properties:
+        is:
+          type: string
+          example: "23202"
+          description: MCC and MNC IS equal to this specific value.
+    MccMncIsInCriteriaItem:
+      type: object
+      properties:
+        is_in:
+          type: string
+          example: "32452, 34567, 56748, 45628"
+          description: MCC and MNC IS IN this list.
+    IsAfterCriteriaItem:
+      type: object
+      properties:
+        is_after:
+          type: string
+          example: "2022-01-21"
+          description: provide date
+    IsBeforeCriteriaItem:
+      type: object
+      properties:
+        is_before:
+          type: string
+          example: "2022-01-21"
+          description: provide date
+    CriteriaResponse:
+      type: "array"
+      items:
+        type: "string"
+      example:
+        - "validNumberFormat"
+        - "numberType"
+        - "activeNumber"
+        - "operatorName"
+        - "operatorCountry"
+        - "mcc"
+        - "mnc"
+        - "disposableNumber"
+    Responses:
+      type: "array"
+      items:
+        type: "string"
+      example:
+        - "operatorName"
+        - "operatorCountry"
+        - "mcc"
+        - "mnc"
+    healthResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          example: "up"
+        updatedAt:
+          type: integer
+          example: 12321546464
+          description: Timestamp
+    versionResponse:
+      type: object
+      properties:
+        version:
+          type: string
+          example: "v1"
+    error400:
+      type: object
+      properties:
+        title:
+          type: string
+          example: "400 Bad Request"
+          description: Error of bad request
+        status:
+          type: integer
+          example: 400
+          description: HTTP error code
+        detail:
+          type: string
+          example: "Invalid country(ies) code(s) provided"
+          description: Error message of bad request
+    error401:
+      type: object
+      properties:
+        message:
+          type: string
+          example: "No API key found in request"
+          description: Unauthorized
+    error403:
+      type: object
+      properties:
+        message:
+          type: string
+          example: "You cannot consume this service"
+          description: Forbidden
+    error404:
```

Comment:
404 error comes in both scenarios: 
- when the requested path does not exist
- when the Template name does not exist (we can mention this as well in the document)

Reply:
example is just a text. It doesn't matter what text we reply back. it could be something like "some message". So this comment i will resolve

Reply:
sure